### ISSUE TITLE
VSCode CloudCode Run/debug w/ Skaffold shows error "expected a single document in the stream, but found more"

### ISSUE THREAD

**Issue by dhodun** [GoogleCloudPlatform/cloud-code-vscode#424]
It seems that CloudCode Skaffold debug does not support multiple documents in the main skaffold.yaml file (wherea skaffold CLI supports this).

**Version information**

Cloud Code Extension version: 1.11.0

VSCode version: 1.55.2

OS: Macos

Cloud SDK:

Google Cloud SDK 331.0.0
bq 2.0.65
core 2021.03.05
gsutil 4.59

Skaffold: 1.23.0

Kubectl:

Client Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.4-dirty", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"dirty", BuildDate:"2021-03-15T09:58:13Z", GoVersion:"go1.16.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.16-gke.502", GitCommit:"a2a88ab32201dca596d0cdb116bbba3f765ebd36", GitTreeState:"clean", BuildDate:"2021-03-08T22:06:24Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}

**Description**: Cloud Code + VSCode skaffold debug on Kubernetes seems not to support a root skaffold.yaml with multiple yaml documents in it.

We currently use 2 different documents/configs to separate our main app services and our ml services. We can run the entire app from CLI with `skaffold debug`, or one half of the app with `skaffold debug -m ml` or `skaffold debug -m backends` respectively.

**Repro step**:
Have a single skaffold.yaml with two documents/configs, separated by `---` then try and run "debug on Kubernetes"

skaffold.yaml :

```yaml
# Master Skaffold
apiVersion: skaffold/v2beta12
kind: Config
metadata:
  name: backends
requires:
  - path: ./microservices/authentication
  configs: [authentication]
  activeProfiles: &dev_and_custom_backends
  - name: custom
    activatedBy: [custom]
  - name: dev
    activatedBy: [dev]
profiles:
- name: custom
- name: dev
  activation:
    - kubeContext: our_cluster
 ---
# Master Skaffold
apiVersion: skaffold/v2beta12
kind: Config
metadata:
  name: ml
requires:
- path: ./microservices/dialog_systems
  configs: [dialog_systems]
  activeProfiles: &dev_and_custom_ml
  - name: custom
    activatedBy: [custom]
  - name: dev
    activatedBy: [dev]
profiles:
- name: custom
- name: dev
  activation:
    - kubeContext: our_cluster

```

**Comment by kschaab** [comment]
Yes, at the moment we don't support this.  We are working on it.

**Comment by kschaab** [comment]
@dhodun your gcloud version is out of date.  It should automatically update as long as you have dependencies automatically managed.  Our extension updates skaffold with gcloud so once this update completes you will have skaffold module support.  I'm not sure what is going on here as far as why gcloud is not updating, but I think this is your actual problem.  If you can turn on verbose logging in the cloud code settings it will give a lot more detail along with error information from the cloud sdk.

**Comment by kschaab** [comment]
This is a bug in our extension, thanks for reporting it.

**Comment by kschaab** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-vscode-internal](https://github.com/GoogleCloudPlatform/cloud-code-vscode-internal/issues/4539)

**Comment by sivakku** [comment]
The fix for this will be part of the May release, we will notify when we put in for insider build.

**Comment by sivakku** [comment]
The fix for this will be part of the May release, we will notify when we put in for insider build.

**Comment by sivakku** [comment]
The fix for this will be part of the May release, we will notify when we put in for insider build.

**Comment by sivakku** [comment]
The fix for this will be part of the May release, we will notify when we put in for insider build.

**Comment by j-windsor** [comment]
You should now see this fixed in Cloud Code 1.12.0. Please reopen this issue if you are still seeing your issue on the latest version.
